### PR TITLE
New post about Csters event

### PR DESCRIPTION
**Pull request by mokaixu**

### CODE REVIEW

File: content/pages/volunteers.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Title: Volunteer
 
 **You can sign up for most volunteer positions by filling out [this
-form](http://goo.gl/forms/NsOXea6Yjf5t02Oj1).**
+form](https://lists.uwaterloo.ca/mailman/listinfo/wics-volunteers).**
```
Comment (evykassirer) [inline content/pages/volunteers.md:4]: same issue with this old stuff (and you still have the file for the options panel in this diff too)

**Comment by arshiamufti** [comment]
Just a question--why are the changes to volunteers.md in this PR? Generally, we like to separate PRs by their purpose.

**Comment by evykassirer** [comment]
@arshiamufti I don't think it's on purpose. there's another PR up for that. she's just getting used to git :)

File: content/W2017/W17-event-options-panel.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+The Options that will be discussed include:
++  Bioinformatics
++  Digital Hardware
++  Software Engineering
++  Business
++  Human-Machine Interaction
++  Computational Fine Arts
++  Computing Technology
++  Human-Computer Interaction
```
Comment (annalorimer) [inline content/W2017/W17-event-options-panel.md:25]: Is this the same as Human-Machine Interaction (line 22)?

**Comment by annalorimer** [comment]
You have an empty file (W17-event-options-panel.md) that needs to be deleted :)

**Pull request closed without merging**